server/src/grpc/query.rs

```diff
@@ -168,7 +168,10 @@ pub async fn fetch_query_output<C: CatalogManager + 'static, Q: QueryExecutor +
 fn convert_output(output: &Output) -> Result<QueryResponse> {
     match output {
         Output::Records(records) => convert_records(records),
-        _ => unreachable!(),
+        Output::AffectedRows(rows) => Err(ServerError::ErrNoCause {
```

Comment:
It seems a little strange to return this as error

Reply:
Yes. There are two reasons.
- The execution error is also a kind of execution result, which needs to be tracked in the test case.
- Our protocol currently doesn't support this field. This is a workaround and I'll file an issue for this.

This is definitely weird and should be fixed soon.

Reply:
Filed https://github.com/CeresDB/ceresdb/issues/89 for this. To my understanding the issue doesn't block this PR? cc @ShiKaiWi may have the same question